src/components/hooks/use-website-status.tsx

```diff
@@ -44,25 +45,39 @@ export const useWebsiteStatusApi = () => {
     // For proper redirection process, this api call will give us the accurate clients's ip address
     // Due to the flexibility to change client's country code via endpoint, it's messing up the redirection flow
     // Therefore we need a direct call from the API
-    const [website_status_api, setWebsiteStatusApi] = useState(null)
+    const [website_status_api, setWebsiteStatusApi] = useState(getAltWebsiteStatus())
 
     useLayoutEffect(() => {
-        const binary_socket = BinarySocketBase.init()
-        binary_socket.onopen = () => {
-            binary_socket.send(JSON.stringify({ website_status: 1 }))
-        }
+        if (!website_status_api) {
+            const binary_socket = BinarySocketBase.init()
+            binary_socket.onopen = () => {
+                binary_socket.send(JSON.stringify({ website_status: 1 }))
+            }
 
-        binary_socket.onmessage = (msg) => {
-            const response = JSON.parse(msg.data)
+            binary_socket.onmessage = (msg) => {
+                const response = JSON.parse(msg.data)
```

Comment:
Let's have a fallback when the `msg.data` is empty

Reply:
Hello @habib-deriv,

This was existing code, and in this case, they keep on trying to send the websocket connection until it's not null. This is what we are running away from. The state's initial value is set to null by default as well  therefore, the only fall back would be to hardcode an object containing a default client's country. Would that be fine?

Reply:
No worries then. Let's keep it as it is..